docs/command-reference/project/create.md

```diff
@@ -31,10 +31,12 @@ lmctl project create [OPTIONS] [LOCATION]
 
 Params for the root Project take the format "param_name param_value", whilst params for Subprojects (those set on `--contains`) use the format "subproject_name.param_name param_value". 
 
-For example, the following sets the driver param to "ansible" for the root Project but "sol003" for the Subproject named A:
+For example, the following sets the driver param to "ansible" for the root Project but "sol003" & "sol005" for the Subproject named A:
```

Comment:
This update is not required as this is just an example (so can leave it as just sol003)